### DISCUSSION TITLE
Maybe a silly question but is the "proxy"  supposed to be the actual app or the "service?"

### DISCUSSION THREAD

**Discussion by Plaidmustache** [Budibase/budibase#15141] [Help]
I deployed budibase on coolify V4 and am able to access the app via the "proxy" link but the "service" link is coming up as a 502. have i mixed something up with the container names, or is it supposed to work like that?  app is deployed at https://budibase-wccgookc8wsc88o00ks0kcgc.flolab.cc/  which was the default link it generated for my instance.

**Comment by mike12345567** [comment]
Hi @Plaidmustache - you connect to Budibase through the proxy service that will be running - you don't communicate with the services directly.
